tool/tctl/common/autoupdate_command.go

```diff
@@ -242,18 +257,174 @@ func (c *AutoUpdateCommand) agentsStatusCommand(ctx context.Context, client auto
 	return nil
 }
 
+func (c *AutoUpdateCommand) agentsReportCommand(ctx context.Context, client autoupdateClient) error {
+	now := c.now()
+	reports, err := getAllReports(ctx, client)
+	if err != nil {
+		if !trace.IsNotFound(err) {
+			return trace.Wrap(err, "listing reports")
+		}
+
+		fmt.Fprintln(c.stdout, "No autoupdate_agent_report found.")
+		if c.ccf != nil && len(c.ccf.AuthServerAddr) > 0 && !strings.Contains(c.ccf.AuthServerAddr[0], "teleport.sh") {
```

Comment:
Might be worth parsing the address and verifying the domain suffix. Technically this will match, e.g., `teleport.ships.io`

Reply:
I thought about this and would like to reuse the existing ping response from when we build the client. We are already doing many round-trips to Teleport and this would make the process even slower.

For now I will use the domain suffix and leave a TODO to refactor tctl so we have access to the ping. (I don't want to do the refactor here as we will backport this PR to v18 and it should be well-contained).